### ISSUE TITLE
The line number area becomes wider every time I change the theme

### ISSUE THREAD

**Issue by ZjzMisaka** [roslynpad/roslynpad#439]
**Describe the bug**
The line number area becomes wider every time I change the theme. 
code:
``` csharp
        private void ActualApplicationThemeChanged(ThemeManager themeManager, object obj)
        {
            if (ThemeManager.Current.ActualApplicationTheme == ApplicationTheme.Light)
            {
                lightModeDocumentId = editor.Initialize(_host, new ClassificationHighlightColors(),
                    Directory.GetCurrentDirectory(), string.Empty);
                documentViewModel.Initialize(lightModeDocumentId);
            }
            else if (ThemeManager.Current.ActualApplicationTheme == ApplicationTheme.Dark)
            {
                darkModeDocumentId = editor.Initialize(_host, new DarkModeHighlightColors(),
                    Directory.GetCurrentDirectory(), string.Empty);
                documentViewModel.Initialize(darkModeDocumentId);
            }
        }
```
![image](https://user-images.githubusercontent.com/16731853/180625560-1556da94-1dd0-49e3-86f6-75c3bc21320f.png)
![image](https://user-images.githubusercontent.com/16731853/180625566-c67722f4-9ddd-4927-beac-ab4ce046d829.png)

I've tried calling ```editor.Initialize``` only once for each theme, and saving the DocumentId, but it doesn't work when used in ```documentViewModel.Initialize```. 

The dark mode based on https://github.com/roslynpad/roslynpad/pull/372 and his sample https://gist.github.com/gmanny/d40f71f0e38a6a9da673a2e05117b847

**To Reproduce**

**Expected behavior**
No matter how many times the theme is switched, the dotted line in the line number area is close to the number. 

**Version**
- OS: Windows 10
- Build (or commit ID if built from source): v3.6.0 downloaded on nuget

**Comment by aelij** [comment]
No longer relevant as WPF was sunset. See the [latest release](https://github.com/roslynpad/roslynpad/releases/tag/22).
